ql/methods/finitedifferences/operators/ninepointlinearop.hpp

```diff
@@ -39,7 +39,7 @@ namespace QuantLib {
         NinePointLinearOp(Size d0, Size d1,
                 const boost::shared_ptr<FdmMesher>& mesher);
         NinePointLinearOp(const NinePointLinearOp& m);
-        NinePointLinearOp(const Disposable<NinePointLinearOp>& m);
+        explicit NinePointLinearOp(const Disposable<NinePointLinearOp>& m);
```

Comment:
This was a false positive. Codacy can't know it, but constructor taking `Disposable`s should allow implicit conversion.